### PR TITLE
Apply changes from trac ticket 103.

### PR DESCRIPTION
**Pull request by mo-g**
See http://cf-trac.llnl.gov/trac/ticket/103

@rhattersley Opinion of my interpretation?

### CODE REVIEW

**Comment by rhattersley** [comment]
Thanks @mo-g ... looks very promising. Would you mind creating a back-link on the trac ticket (e.g. as at http://cf-trac.llnl.gov/trac/ticket/71#comment:4).

File: appa.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 All CF attributes are listed here except for those that are used to describe grid mappings. See Appendix F for the grid mapping attributes.
 
-The "Type" values are **S** for string, **N** for numeric, and **D** for the type of the data variable. The "Use" values are **G** for global, **C** for variables containing coordinate data, and **D** for variables containing non-coordinate data. "Links" indicates the location of the attribute"s original definition (first link) and sections where the attribute is discussed in this document (additional links as necessary).
+The "Type" values are **S** for string, **N** for numeric, and **D** for the type of the data variable. The "Use" values are **G** for global, **C** for variables containing coordinate data, **D** for variables containing non-coordinate data and **-** for variables with a special purpose. "Links" indicates the location of the attribute"s original definition (first link) and sections where the attribute is discussed in this document (additional links as necessary).
```
Comment (rhattersley) [inline appa.adoc]: It'd be nice to preserve the [serial comma](https://en.wikipedia.org/wiki/Serial_comma) to match the previous style:

    ... non-coordinate data, and **-** for variables with a special purpose.

**Comment by rhattersley** [comment]
Other than the *tiny* issue of a single comma ... all good. :+1:

**Comment by mo-g** [comment]
Pushed the comma change, applied for a trac account.

Cheers,
Gray.

**Comment by rhattersley** [comment]
Super, thanks @mo-g. :+1:

I'll merge this once #71 has been merged.

**Comment by rhattersley** [comment]
Sorry, I forgot that there's one more thing to do: the change needs a corresponding addition to the "Revision History" appendix. I'm trying to clarify (via a separate e-mail thread) what date to use, but if you're keen then I'd suggest starting with the date you applied the changes (i.e. now-ish), rather than the dates from the trac ticket.

**Comment by rhattersley** [comment]
> I'm trying to clarify (via a separate e-mail thread) what date to use, but if you're keen then I'd suggest starting with the date you applied the changes (i.e. now-ish), rather than the dates from the trac ticket.

Please do use the date you applied the changes. I've had clarification from @JonathanGregory:

> Yes, the date when the change was applied would be fine.

**Comment by mo-g** [comment]
Rebased to current master. ;)

File: appg.adoc [hunk 2]
```diff
@@ -93,3 +93,6 @@ Re-do several changes which had previously been made in an earlier draft of vers
 .20 January 2016
 . link:$$http://cf-trac.llnl.gov/trac/ticket/67$$[Ticket 67], remove
 deprecation of "missing_value" from <<attribute-appendix>>.
+
+.29 February 2016
+In Appendix A (Attributes), updated Type and Use values for some attributes and added "special purpose" value. In Appendix H (Annotated Examples of Descrete Geometries), updated coordinate values for the variables in some examples to correct omissions.
```
Comment (rhattersley) [inline appg.adoc]: Please could you:
1. Add a link to the trac ticket (see previous entry for ticket 67 for an example).
2. Use cross-references to refer to the modified appendices, i.e. `<<attribute-appendix>>` and `<<appendix-examples-discrete-geometries>>`.

**Comment by mo-g** [comment]
Hi Richard,

I think this is ready to merge. Are you happy?

Cheers,
Gray.

**Comment by rhattersley** [comment]
Thanks @mo-g :+1:

**Pull request merged**